CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.0.0
+
+* turf-grid => turf-point-grid (turf.pointGrid)
+* turf-hex => turf-hex-grid (turf.hexGrid)
+* turf-hex-grid now has a required `unit` parameter
+* remove turf-isobands
```

Comment:
can you add a hint on how one would upgrade? like "if you were using isobands, now use isolines?"